searchlib/src/vespa/searchlib/query/streaming/weighted_set_term.cpp

```diff
@@ -18,8 +18,9 @@ WeightedSetTerm::WeightedSetTerm(std::unique_ptr<QueryNodeResultBase> result_bas
 WeightedSetTerm::~WeightedSetTerm() = default;
 
 void
-WeightedSetTerm::unpack_match_data(uint32_t docid, const ITermData& td, MatchData& match_data)
+WeightedSetTerm::unpack_match_data(uint32_t docid, const ITermData& td, MatchData& match_data, const fef::IIndexEnvironment& index_env)
 {
+    (void) index_env;
```

Comment:
When the type is specific you do not need a name for the parameter, and then you do not need the explicit (void).